back/engines/commercial/analysis/spec/acceptance/inputs_spec.rb

```diff
@@ -99,6 +100,16 @@
         })
       end
 
+      example 'supports limit', document: false do
+        create(:idea, title_multiloc: { en: 'Idea one' }, project: analysis.source_project)
+        create(:idea, body_multiloc: { en: 'Idea two' }, project: analysis.source_project)
+        do_request(limit: 1)
+        expect(status).to eq(200)
+        expect(json_response_body[:meta]).to match({
+          filtered_count: 1
+        })
```

Comment:
I think we should also add a check that only one instance is returned? Something like:
```
expect(response_data.size).to eq 1
```